docs/docsite/rst/community/other_tools_and_programs.rst

```diff
@@ -1,24 +1,108 @@
-************************
+########################
 Other Tools And Programs
-************************
+########################
 
-The Ansible community provides several useful tools for working with the Ansible project. This is a list
-of some of the most popular of these tools.
+.. contents:: Topics
 
-- `PR by File <https://ansible.sivel.net/pr/byfile.html>`_ shows a current list of all open pull requests by individual file. An essential tool for Ansible module maintainers.
+The Ansible community provides several useful tools for working with the Ansible project. This is a list of some of the most popular of these tools.
+
+*******
+Editors
+*******
+
+Atom
+====
+
+Is a open source and free GUI text editor by GitHub. It can keep track of git project
+changes, can commit from the GUI and see what branch you are on. You can customize the
+themes for different colors and install packages to help with different languages Language
+YAML. You can install it on Linux, OSX and Windows.
+
+* `Language YAML <https://github.com/atom/language-yaml>`_
+
+
+Pycharm
+=======
+
+Is a full IDE to develop Python software. It has all the needed parts to be able to write
+python scripts and complete software. Is a little overkill for writing roles/playbooks but, it can
+be a very useful tool if you write modules and submit code for Ansible.
```

Comment:
Is it worth adding a comment saying you can use PyCharm to debug the engine compontents or is designed to be module specific?

Reply:
Meant to be generic, good call. Updated.